Assets/Scripts/Entity/Enemy/Boss/PhaseTwo.cs

```diff
@@ -51,7 +51,7 @@ private void Update()
         // Meteor attack
         if (phaseTwoAttackTimer > timeToPhaseTwoAttack)
         {
-            Instantiate(phaseTwoAttack, gameObject.transform);
+            Instantiate(phaseTwoAttack, GameObject.FindGameObjectWithTag("Player").transform);
```

Comment:
Note this will make the `phaseTwoAttack` a child of the player. Which also means that it will move when  the player moves, which may not have been intended.

If you meant to spawn it at the position of the player, then you must use the 4th overloaded function for [Instantiate](https://docs.unity3d.com/ScriptReference/Object.Instantiate.html): `public static Object Instantiate(Object original, Vector3 position, Quaternion rotation);`